roles/privacyidea_ha/tasks/import_data.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Copy the privacyidea_data_import_and_cleanup script to /etc/privacyidea
+  template:
+    src: privacyidea_data_import_and_cleanup.sh
+    dest: /etc/privacyidea/privacyidea_data_import_and_cleanup.sh
+    owner: www-data
+    group: www-data
+    mode: '0755'
+
+# - name: Execute the privacyidea_data_import_and_cleanup.sh script
```

Comment:
why is that not included?

Reply:
As this does not work automaticly and requres data to be provided.